packages/auth-compat/test/integration/flows/custom.test.ts

```diff
@@ -137,6 +140,65 @@ describe('Integration test: custom auth', () => {
     expect(customUser!.uid).not.to.eql(anonUser!.uid);
   });
 
+  context('in passthrough mode', () => {
+    beforeEach(async () => {
+      const updatedConfig = await updateEmulatorProjectConfig(
+        JSON.stringify({
+          usageMode: 'PASSTHROUGH'
+        })
+      );
+      expect(updatedConfig).to.eql({
+        signIn: { allowDuplicateEmails: false },
+        usageMode: 'PASSTHROUGH'
+      });
+    });
+
+    afterEach(async () => {
+      await updateEmulatorProjectConfig(
+        JSON.stringify({
+          usageMode: 'DEFAULT'
+        })
+      );
+    });
+
+    xit('signs in with custom token in passthrough mode', async () => {
+      const cred = await firebase.auth().signInWithCustomToken(customToken);
+      expect(firebase.auth().currentUser).to.eq(cred.user);
+      expect(cred.operationType).to.eq('signIn');
+
+      const { user } = cred;
```

Comment:
nit: do this instead to avoid having to assert user every time on the lines below

```ts
const user = cred.user!;
```